packages/rspack/src/builtin-plugin/css-extract/loader.ts

```diff
@@ -79,10 +84,15 @@ export const pitch: LoaderDefinition["pitch"] = function (request, _, data) {
 		this._compiler &&
 		this._compiler.options &&
 		this._compiler.options.experiments &&
-		this._compiler.options.experiments.css
+		this._compiler.options.experiments.css &&
+		this._module &&
+		(this._module.type === "css" ||
+			this._module.type === "css/auto" ||
+			this._module.type === "css/global" ||
+			this._module.type === "css/module")
 	) {
 		let e = new Error(
-			"You can't use `experiments.css` and `css-extract-rspack-plugin` together, please set `experiments.css` to `false`"
+			'You can\'t use `experiments.css` (`experiments.futureDefaults` enable built-in CSS support by default) and `CssExtractRspackPlugin` together, please set `experiments.css` to `false` or set `{ type: "javascript/auto" }` for rules with `CssExtractRspackPlugin` in your webpack config (now `CssExtractRspackPlugin` does nothing).'
```

Comment:
> 'You can\'t use `experiments.css` (`experiments.futureDefaults` enable built-in CSS support by default) and `CssExtractRspackPlugin` together,

I thinks this is inaccurate, since we did use experiments.css and cssExtractRspackPlugin together in the test case, what you mean maybe "You can't use type:css with cssExtractRspackPlugin together" and experiments.css set type:css for css file by default so remember set type:"javascript/auto" for css file if you want to use cssExtractRspackPlugin to handle css files

Reply:
Done